CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+- Handle case where a prefix is used and no objects are found. (@akatch)
```

Comment:
Per our [changelog conventions](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) this would go under a `### Fixed` header underneath `## [Unreleased]`